config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-presubmits.yaml

```diff
@@ -16,6 +16,7 @@ presubmits:
         args:
         - "--job=$(JOB_NAME)"
         - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
```

Comment:
Hi @figo,

Where did you find the flag for `--root`? I don't see it mentioned at https://github.com/kubernetes/test-infra/tree/master/prow.

Assuming `--root` is correct, I question its value. The path to which the repo must be cloned is `$GOPATH/src/k8s.io/cloud-provider-vsphere`. Does setting the `root` value to `/go/src` result in that? If anything I would think a `--root=/go/src` would result in sources at either directly in `/go/src` or `/go/src/cloud-provider-vsphere`. Both of which are still invalid.

Reply:
i am referring from: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L535, i think the working folder will be `--root` + `--repo`, which is `/go/src/k8s.io/cloud-provider-vsphere`, thanks

Reply:
yeah, this isn't documented outside of `jenkins/bootstrap.py` (which is what ultimately consumes this flag) because we want to kill that tool entirely.

Reply:
if / when we get to that point we'll migrate any existing jobs. the new tooling has better docs but isn't quite to feature parity. we're still figuring that out.